rasa_utils/bot_server_channel.py

```diff
@@ -150,10 +150,10 @@ def tracker(cid):
 
         @custom_webhook.route("/conversations/<cid>/say", methods=["GET"])
         def say(cid):
-            message = request.args.get(b"message", [])
-            _payload = request.args.get(b"payload", [])
-            _display_name = request.args.get(b"display_name", [])
-            _uuid = request.args.get(b"uuid", [])
+            message = bytes(request.args.get("message"), "utf8")
+            _payload = bytes(request.args.get("payload", ""), "utf8")
```

Comment:
Is wrapping everything in `bytes` necessary? What is the purpose of that?

Reply:
I added it to replace `b"message"`, which produces a bytes instance instead of a string. The new request from Flask doesn't seem to like the format of `b"message"`, and without converting the argument to bytes, it would error at `{"type": "text", "text": message.decode("utf-8")}`. I can make a change if there's a better way of handling this 🙂

Reply:
Fair enough. Just wanted to understand what's going on :-) Leave it as is then.